modules/ROOT/partials/configuration/newline_behavior.adoc

```diff
@@ -0,0 +1,27 @@
+[[newline_behavior]]
+== `+newline_behavior+`
+
+NOTE: This option is only available for TinyMCE 6.1 and later.
+
+This option enables you to adjust what happens when enter or return is pressed. There are four options:
```

Comment:
`you` is a pronoun is something we should avoid as per the style guide: https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/CONTRIBUTING.md#pronouns

```suggestion
This option enables adjusting what happens when the enter or return key is pressed. There are four options:
```

Reply:
This is literally copy/paste from forced_root_block with some adjustments. There are 112 instances of `enables you`, I feel like they would be better changed all at once... for consistency if nothing else.